docs/api/app.md

```diff
@@ -147,18 +166,19 @@ A dispatch initializer that can create a [custom dispatch function](../architect
 ```js
 app({
   init: { message: "Hello, World!" },
-  view: (state) => h("p", {}, text(state.message)),
+  view: (state) => h("body", {}, h("p", {}, text(state.message))),
   node: document.body,
 })
 ```
-<!-- A "Hello, World!" program is traditionally the first program you would write when learning a new programming language. -->
+
+<!-- A “Hello, World!” program is traditionally the first program you would write when learning a new programming language. -->
 
 ### Full Usage
 
 ```js
 app({
   init: { message: "Hello, World!" },
-  view: (state) => h("p", {}, text(state.message)),
+  view: (state) => h("body", {}, h("p", {}, text(state.message))),
```

Comment:
Same here and everywhere else. 🙏

Reply:
I don't quite get your comment. Did I miss something (and need to correct) or do you like my change?